lib/phoenix_component.ex

```diff
@@ -1073,15 +1073,22 @@ defmodule Phoenix.Component do
         """
       end
 
-  ## Referencing parent assigns
+  ## Connected and disconnected state
```

Comment:
I think the previous title was better because we are still mostly talking about assigns. Perhaps we keep `## Referencing parent assigns` and split into `### Disconnected state` and `### Connected state`? Or maybe call it `## Sharing assigns`. WDYT?

Reply:
Sharing assigns seems to me better because there is no parent assign when the liveview starts 
Referencing parent assigns is only in the context of nested liveview, indeed conn is not a parent assign but a previous and disjoint assign

Reply:
Yeah, let’s go that way then!